docs/en/latest/admin-api.md

```diff
@@ -547,11 +547,11 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 `hash_on` can be set to different types:
 
 1. when it is `vars`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html), without the prefix '$'.
-1. when it is `header`, the `key` is required. It is equal to "http_`key`".
-1. when it is `cookie`, the `key` is required. It is equal to "cookie_`key`".
-1. when it is `consumer`, the `key` is optional. The key is the `consumer_name` set by authentication plugin.
-1. when it is `vars_combinations`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html) combinations, such as `$request_uri$remote_addr`.
-1. If there is no value for either `hash_on` or `key`, `remote_addr` will be used as key.
+2. when it is `header`, the `key` is required. It is equal to "http_`key`".
```

Comment:
We don't need to change this. Markdown will handle the order automatically. There is no need to maintain the order by us.

Reply:
yes, fixed